doc/flux_to_redux.md

```diff
@@ -0,0 +1,641 @@
+
+1. __Diagram store and component relationships.__ Similar to the Advanced conversion, it still makes sense to understand the current state of the architecture. For Influx in particular, the focus should be on revealing what stores the specific components are reliant on. In some cases, stores may also be reliant on other stores, which should also be noted. At the end of this step, the goal is to be able to point to any component and easily understand all the stores that component relies on.
+1. __Break the refactor into groups that includes the stateful layer and the UI layer.__ This will also be like Advanced conversion. The difference is that coming out of this step, stores and the UI should be broken into tangible groups to be refactored. These groups will be the core of the refactor plan. Ideally, each group should be independent enough that it can be updated without touching any other group. The groups should also be as small as possible, while also making sense, to allow for the most incremental update possible. If diagraming the library architecture (during the previous step) went well, this should be as easy as looking at the diagram and noting the groups that emerge.
+
+#### Phase 2: Incrementally Update Stores
+
+__Goal:__ Update all stores to be either Influx or Redux and update all components to Redux connected components. This phase can be done incrementally, and should be repeated for every store or every group of tightly coupled stores. Consequently, the steps should be read through the lens of that specific store or group of stores.
+
+1. __Refactor relevant stores to be flat.__ State should be lifted up or broken out into their own instances so that stores are not nested. Redux holds firmly that an application should have only one store. However, while refactoring to a single store Redux architecture, Influx can have multiple stores. The rule is that stores cannot be nested.
+1. __Decide which stores will be Redux and which stores will be Influx.__ An Influx implementation can have both Redux stores and Influx stores. The rules are:
+    - A Redux component can talk to both a Redux store and an adapted Influx store at the same time.
+    - A Flux component can talk to an Influx store and a Flux store at the same time.
+    - A `connectFlux` component can talk to an adapted Influx store.
+
+    The result of these rules is that a store can be converted directly to Redux [iff](https://en.wikipedia.org/wiki/If_and_only_if) it will be used exclusively by Redux components, but a component can be converted to a Redux component if it will pull data from either an adapted Influx store or a traditional Flux store.
+
+    If the stores are not overly complex and all of the related UI can be updated to Redux connected components (extending from `UiComponent2`), then going straight to Redux may be a good option.
```

Comment:
some of the verbage around which piece to convert when and how to decide reads as a bit circuitous - could some of the prose be eliminated by using a simple flowchart / decision tree?

Reply:
Yep, good call! I added that and reworded it a little bit